### ISSUE TITLE
How to set output fps？

### ISSUE THREAD

**Issue by aliofwind** [rigaya/NVEnc#89]
My video is 60fps, I need to encode to 30fps, how to set output fps for nvenc?

**Comment by rigaya** [comment]
Unfortunately, NVEncC does not support frame rate conversion. You'll need to do it via Avisynth/VapourSy nth.

**Comment by parhamsan** [comment]
FYI,
https://forum.videohelp.com/threads/322014-how-to-change-frame-rate-on-staxrip

**Comment by aliofwind** [comment]
thanks, but Avisynth with SelectEven() used too many cpu resources, x264 is more cost effective

**Comment by rigaya** [comment]
I've added option for reducing framerate in [NVEncC 4.23](https://github.com/rigaya/NVEnc/releases/tag/4.23), please try using [--vpp-select-every](https://github.com/rigaya/NVEnc/blob/master/NVEncC_Options.en.md#--vpp-select-every-intparam1int).

```
example1 (same as "select even"): --vpp-select-every 2
example2 (same as "select odd "): --vpp-select-every 2,offset=1
```

**Comment by aliofwind** [comment]
Thank you so much, it saves a lot of time and resources

**Comment by aliofwind** [comment]
But the bitrate in NvEncC's log is the double of actual bitrate
`encoded 8037 frames, 77.55 fps, 49324.75 kbps, 787.62 MB`
(log)
> Duration : 4 min 27 s
Bit rate : 24.7 Mb/s
Frame rate : 30.000 FPS
Stream size : 788 MiB (99%)

(MediaInfo)

**Comment by rigaya** [comment]
Thanks, the problem with the bitrate was fixed on NVEnc 4.24.
